### PR TITLE
Dev terminal

### PR DESCRIPTION
**Pull request by SankeerthTella**
Added terminal for SFTP login and able execute ssh commands. As of improvement backend code should be implemented using web sockets.

### CODE REVIEW

**Review by aashish-jain — APPROVED** [review]
Changes look good. @jgoldverg check it once and merge it to dev.

File: pom.xml [hunk 1]
```diff
@@ -228,6 +228,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <version>0.1.55</version>
```
Comment (jgoldverg) [inline pom.xml:234]: [WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: com.jcraft:jsch:jar -> version 0.1.54 vs 0.1.55 @ line 231, column 21. 
This new warning pops up that can break our build potentially can we upgrade the version to the LTS one?

File: src/main/java/org/onedatashare/server/controller/SwaggerController.java [hunk 2]
```diff
@@ -47,9 +47,9 @@
 @RestController
 @Slf4j
 public class SwaggerController {
-
 //    @RequestMapping(value = "swagger-ui.html", method = RequestMethod.GET)
 //    public void getSwagger(HttpServletResponse httpResponse) throws IOException {
 //        httpResponse.setStatus(HttpStatus.INTERNAL_SERVER_ERROR.value());
 //    }
+
```
Comment (jgoldverg) [inline src/main/java/org/onedatashare/server/controller/SwaggerController.java]: Can you delete this file from the PR? It is not needed.

File: src/main/java/org/onedatashare/server/service/SSHConsoleService.java [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        catch (Exception e){
+            e.printStackTrace();
+            return Flux.error(new Exception("Error while Executing command"));
+        }
+    }
+
+}
+
+
+class SSHUserInfo implements UserInfo{
```
Comment (jgoldverg) [inline src/main/java/org/onedatashare/server/service/SSHConsoleService.java]: Can we put the SSHUserInfo class into its own file? It is bad practice to have these classes laying around not in their own files unless there is a specific dependency. Java has a standard of each class in its own file.

File: src/main/java/org/onedatashare/server/service/SSHConsoleService.java [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        });
+    }    //createSession()
+
+    public Flux runCommand(UserAction ua, String commandWithPath){
+        return createSession(ua.getUri(), ua.getCredential().getUsername(), ua.getCredential().getPassword(), Integer.parseInt(ua.getPortNumber()))
+                .flux()
+                .flatMap(session -> {
+                    try {
+                        session.connect();
+                        ChannelExec channel = (ChannelExec)session.openChannel("exec");
```
Comment (jgoldverg) [inline src/main/java/org/onedatashare/server/service/SSHConsoleService.java:46]: Is it ok that "exec" is a hardcoded value for our channel's?

File: src/main/java/org/onedatashare/server/service/SSHConsoleService.java (hunk 3 above)
Comment (aashish-jain) [reply src/main/java/org/onedatashare/server/service/SSHConsoleService.java]: Done

**Pull request merged**